### PR TITLE
Indicate that using `bundle exec` is no longer recommended as of v0.7.0

### PR DESCRIPTION
**Pull request by vinistock**
### Motivation

Starting with v0.7.0, we'll ship with the custom bundle functionality from the server and there's no more need to use `bundle exec` or include the gem in the `Gemfile`.

### CODE REVIEW

File: README.md
Comment (st0012) [inline README.md:31]: I don't completely understand what this means. Could you mean this?

```suggestion
custom bundle in `.ruby-lsp/Gemfile` which is used with the versions of dependencies that should be used (e.g.: RuboCop).
```

**Review by st0012 — COMMENTED** [review]
Should we wait for the release of `v0.7.0` before we merge this?

File: README.md
Comment (andyw8) [inline README.md:31]: Missing a word here I think?
```suggestion
custom bundle in `.ruby-lsp/Gemfile` which is used to specify the versions of dependencies that should be used (e.g.: RuboCop).
```

**Review by andyw8 — CHANGES_REQUESTED** [review]

**Comment by vinistock** [comment]
Yeah, there was a word missing 🤦. Probably a lot easier to understand now, but please let me know if that's not the case.

> Should we wait for the release of v0.7.0 before we merge this?

Given that we mention this applies only to v0.7.0 and up, I think it's okay to merge. Do you prefer to wait?

File: EDITORS.md
Comment (bitwise-aiden) [inline EDITORS.md:15]:
```suggestion
[Eglot](https://github.com/joaotavora/eglot) runs solargraph server by default. To set it up with ruby-lsp you need to put that in your init file:
```

**Review by bitwise-aiden — APPROVED** [review]
Small nit, but lgtm

**Comment by st0012** [comment]
> Given that we mention this applies only to v0.7.0 and up, I think it's okay to merge. Do you prefer to wait?

Would the emacs change work before v0.7.0? If it would, then I'm good with merging it now 👍

**Comment by vinistock** [comment]
That's fair, we don't mention the specific version on the other editors docs. Let's wait a bit until we release v0.7.0.

File: EDITORS.md
Comment (andyw8) [reply EDITORS.md:15]: (it also makes diffs much easier to read)

Comment (vinistock) [reply EDITORS.md:15]: I think one sentence per line might be a bit too much. As long as it's wrapping in 120 characters and it fits on the screen, it should be fine.

**Pull request merged**
